package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "service-template-node",
-  "version": "0.5.8",
+  "version": "0.5.9",
   "description": "A blueprint for MediaWiki REST API services",
   "main": "./app.js",
   "scripts": {
     "start": "service-runner",
-    "test": "PREQ_CONNECT_TIMEOUT=15 mocha && npm run lint",
-    "lint": "eslint --cache --max-warnings 0 --ext .js --ext .json .",
+    "test": "PREQ_CONNECT_TIMEOUT=15 npm run lint && mocha",
```

Comment:
`PREQ_CONNECT_TIMEOUT` won't be picked up here, you need to move it:

```
npm run lint && PREQ_CONNECT_TIMEOUT=15 mocha
```

Reply:
Done